src/Compilers/CSharp/Portable/Symbols/Compilation_WellKnownMembers.cs

```diff
@@ -141,6 +141,38 @@ internal bool IsExceptionType(TypeSymbol type, ref HashSet<DiagnosticInfo> useSi
             return IsEqualOrDerivedFromWellKnownClass(type, WellKnownType.System_Exception, ref useSiteDiagnostics);
         }
 
+        internal bool IsTupleUnderlyingType(TypeSymbol typeToCheck, int requiredArity)
```

Comment:
If we support multiple tuple symbols, this helper would need to be changed to not be in terms of well known types.